modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -252,12 +252,40 @@ definition {
 	}
 
 	@description = "Verify that the user can assign a task to another user in the Assigned to My Roles tab"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAssignTaskToOtherUser {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry Title");
+
+		JSONUser.addUser(
+			userEmailAddress = "usersn@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		JSONRole.assignRoleToUser(
+			roleTitle = "Administrator",
+			userEmailAddress = "usersn@liferay.com");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
 
-		// TODO LRQA-68843 CanAssignTaskToOtherUser pending implementation
+		Workflow.assignByDetails(user = "usersn (userfn userln)");
 
+		Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");
```

Comment:
The step 7 can be extended, with this you are checking the success message, but it's not making a assert if at the new user, there is a blog assigned to him. You can logout and log in to check if the blogs is present there.